mason.py

```diff
@@ -378,6 +378,10 @@ def get_env_vars(
                     beaker.BeakerEnvVar(name="NCCL_SOCKET_IFNAME", value="enp0s12"),
                     # Add COLL here to log all collective operations. Extreamly verbose, dont use for production.
                     beaker.BeakerEnvVar(name="NCCL_DEBUG_SUBSYS", value="INIT,NET"),
+                    beaker.BeakerEnvVar(
+                        name="LD_LIBRARY_PATH", value=f"/var/lib/tcpxo/lib64:{os.getenv(LD_LIBRARY_PATH, '')}"
```

Comment:
LGTM but I'm also seeing these from DPO:
```
    --env NCCL_PROTO=Simple,LL128 \
    --env NCCL_TUNER_CONFIG_PATH=/var/lib/tcpxo/lib64/a3plus_tuner_config_ll128.textproto \
    --env NCCL_SHIMNET_GUEST_CONFIG_CHECKER_CONFIG_FILE=/var/lib/tcpxo/lib64/a3plus_guest_config_ll128.textproto \
```
I'm not sure they're necessary, especially the NCCL_PROTO one, but just flagging for awareness.

We're also running `source /var/lib/tcpxo/lib64/nccl-env-profile.sh` before launching the actual accelerate command.

Reply:
Yes, these are all set (except for `nccl-env-profile.sh`). I'm not sure how to source `nccl-env-profile.sh` programmatically. Do you have any ideas?